src/components/Common/GoHome/GoHome.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { GoHome as GoHomeIcon } from 'react-icons/go';
+import { IoReturnUpBackSharp } from 'react-icons/io5';
+import { Link } from 'react-router-dom';
+
+import styles from 'components/Common/GoHome/GoHome.module.scss';
+
+const GoHome = () => {
```

Comment:
I would suggest you to update this component to `BackButton`

Reply:
and its function will return back, not to home ?

Reply:
yes